### PR TITLE
demo

### PR DESCRIPTION
**Pull request by tarasulo**

### CODE REVIEW

File: serviceToXML/src/main/java/tkhal/service/stringToXMLController/CustomerFactory.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    public static LinkedList<Customer> create(String message) {
+        LinkedList<Customer> list = new LinkedList<>();
+        String[] records = message.split("\n");
+        for (String record : records) {
+            String symbol = record.substring(record.length()-1);
+            if (symbol.equals(",")) {
+                record += "?";
+            }
+            String[] value = record.split(",");
+            if (value[18].equals("\"")) {
```
Comment (kolyansv) [inline serviceToXML/src/main/java/tkhal/service/stringToXMLController/CustomerFactory.java]: 1. hard code is bad idea
2. your solution is not permanent - what if comma will be as a part of value for column different from 18 ?

File: serviceToXML/src/main/java/tkhal/service/stringToXMLController/CustomerFactory.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                if (!(value[4].equals("?"))) customer.setCrossModelId(value[4]);
+                if (!(value[5].equals("?"))) customer.setTailNumber(value[5]);
+                if (!(value[6].equals("?"))) customer.setDepIcaoSiteId(value[6]);
+                if (!(value[7].equals("?"))) customer.setDepIcaoSiteId(value[7]);
+                if (!(value[8].equals("?") || value[8].length() == 0)) try {
+                    customer.setCopyNum(Integer.parseInt(value[8]));
+                } catch (NumberFormatException e) {
+                    LOGGER.error("error in element copyNum" + e);
+                }
+                if (!(value[9].equals("?") || value[9].length() == 0)) try {
```
Comment (kolyansv) [inline serviceToXML/src/main/java/tkhal/service/stringToXMLController/CustomerFactory.java]: a lot of duplicated code. Pay attention on if statements

Comment (tarasulo) [reply serviceToXML/src/main/java/tkhal/service/stringToXMLController/CustomerFactory.java]: made boolean function for general if statements

File: serviceToXML/src/main/java/tkhal/service/stringToXMLController/CustomerFactory.java (hunk 1 above)
Comment (tarasulo) [reply serviceToXML/src/main/java/tkhal/service/stringToXMLController/CustomerFactory.java]: fixed

**Pull request merged**